src/Client.php

```diff
@@ -172,33 +172,28 @@ protected function request($method, $path, array $options = [])
      */
     private function doRequest(RequestInterface $request, array $options)
     {
+        $exceptionMap = [
+            'InvalidQuery' => Exception\InvalidQueryException::class,
+            'AccessTokenInvalid' => Exception\AccessTokenInvalidException::class,
+            'NotFound' => Exception\NotFoundException::class,
+            'RateLimitExceeded' => Exception\RateLimitExceededException::class
+        ];
+
```

Comment:
We could probably move this definition to _a better place_ ™. What do say about maybe having a static `ApiException::getExceptionMap()`?